src/hooks/useVotesRevealedEventsSummary.ts

```diff
@@ -0,0 +1,33 @@
+import { useContext } from "react";
+import { useQuery } from "react-query";
+import {
+  queryVotesRevealedEvents,
+  VoteRevealed,
+} from "web3/get/queryVotesRevealedEvents";
+import { ErrorContext } from "common/context/ErrorContext";
+import provider from "common/utils/web3/createProvider";
+import createVoidSignerVotingContractInstance from "web3/createVoidSignerVotingContractInstance";
+import determineBlockchainNetwork from "web3/helpers/determineBlockchainNetwork";
+
+const contract = createVoidSignerVotingContractInstance(
+  provider,
+  determineBlockchainNetwork()
+);
+
+export default function useVotesRevealedEventsSummary() {
+  const { addError } = useContext(ErrorContext);
+
+  const { data, error, isFetching, refetch } = useQuery<
```

Comment:
Is there a reason why you destructure this and restructure it on the return, rather than just returning the useQuery directly?

Reply:
Nothing to change, but one thing I find useful is to do `const {data, ...others} = useQuery(...)` and then returning them like `{someRelevantKey: data, ...others}`, which helps when using multiple `useQuery` hooks in the same component as you don't have a ton of `data` that you have to rename

Reply:
Yeah, David had suggested earlier we just return the default values then allow for the individual views to just handle naming convention as they see fit. It works either way.

And I often like returning what is currently relevant, as there is a ton of extra stuff useQuery actually returns from it's object. But we can just return the whole thing if we wish.

Reply:
its up to you, its just a style thing, just wanted to get your thoughts. i think its fine any of these ways